src/app/spreed-webrtc-server/random.go

```diff
@@ -23,27 +23,33 @@ package main
 
 import (
 	"crypto/rand"
+	"math/big"
 	pseudoRand "math/rand"
 	"time"
 )
 
 const (
-	dict = "abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVW0123456789"
+	dict = "abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ0123456789"
 )
 
+// NewRandomInt returns a random integer in range [0, max[
+// Tries to use crypto/rand and falls back to math/rand
+func NewRandomInt(max int) int {
+
+	rand, err := rand.Int(rand.Reader, big.NewInt(int64(max)))
```

Comment:
Please pass `max` as `*big.Int` and create it only once in `NewRandomString` below to avoid creating too many temp objects.